src/common/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './Notifications';
 export * from './RootProviders';
 export * from './SecureStore';
 export * from './testID';
+export * from './Analytics';
```

Comment:
Should we avoid exporting `_sdkTracker`?

Reply:
I'm not really sure. I think it would still be exported but the underscore usually just indicates that it is internal. I'm not sure what would be best.